examples/videocomposer/README.md

```diff
@@ -79,6 +81,72 @@ You can adjust the arguemnts in `vc/config/base.py` (lower-priority) or `configs
 - mode: 0 for use graph mode,  1 for pynative mode
 
 
+### Offline (Mindspore Lite) Inference
+
+
+#### Install Mindspore Lite
+You need to have a Mindspore Lite Environment first for offline inference.
+
+To install Mindspore Lite, please Refer to [Lite install](https://mindspore.cn/lite/docs/zh-CN/r2.1/use/downloads.html)
+
+1. Download the supporting tar.gz and whl packages according to the environment.
+2. Unzip the tar.gz package and install the corresponding version of the WHL package.
+
+   ```shell
+   tar -zxvf mindspore-lite-2.1.0-*.tar.gz
+   pip install mindspore_lite-2.1.0-*.whl
+   ```
+
+3. Configure Lite's environment variables
+
+   `LITE_HOME` is the folder path extracted from tar.gz, and it is recommended to use an absolute path.
+
+   ```shell
+   export LITE_HOME=/path/to/mindspore-lite-{version}-{os}-{platform}
+   export LD_LIBRARY_PATH=$LITE_HOME/runtime/lib:$LITE_HOME/tools/converter/lib:$LD_LIBRARY_PATH
+   export PATH=$LITE_HOME/tools/converter/converter:$LITE_HOME/tools/benchmark:$PATH
+
+
+#### Export Mindspore Lite Model
```

Comment:
need to unset some environment variables before lite export? Please make sure all necessary instructions for a successful run are included in readme

Reply:
mendtioned in line 128